cloudinit/version.py

```diff
@@ -4,7 +4,7 @@
 #
 # This file is part of cloud-init. See LICENSE file for license information.
 
-__VERSION__ = "23.4"
```

Comment:
This branch isn't 23.4.1 based it's 23.4 based. we are missing the dbus fixes for early boot from this branch and other downstreams.